ingest-app/src/cmr/ingest/api/core.clj

```diff
@@ -271,7 +273,7 @@
   Don't try to read the body again after calling this function."
   ([concept-type body content-type headers]
    (let [metadata (read-multiple-body! body)]
-     (metadata->concept concept-type metadata content-type headers)))
+     (tee (metadata->concept concept-type metadata content-type headers))))
```

Comment:
Is tee to be deleted.  It looks like debugging code since the function above uses print.